paf-mvp-demo-express/src/crto1-operator.ts

```diff
@@ -39,10 +39,14 @@ export const crtoOneOperatorApp = express();
 // This host supports the Operator API
 addOperatorApi(
   crtoOneOperatorApp,
+  {
+    name: crtoOneOperatorConfig.name,
+    currentPublicKey: operatorPrivateConfig.currentPublicKey,
+    dpoEmailAddress: operatorPrivateConfig.dpoEmailAddress,
+    privacyPolicyUrl: new URL(operatorPrivateConfig.privacyPolicyUrl),
+  },
```

Comment:
I'm sorry I don't understand what you are suggesting, as I don't see where inference is involved.
Would you want to create an Identity variable apart, like so?
```typescript
const identity: Omit<Identity, 'type'> = {
  name: crtoOneOperatorConfig.name,
  currentPublicKey: operatorPrivateConfig.currentPublicKey,
  dpoEmailAddress: operatorPrivateConfig.dpoEmailAddress,
  privacyPolicyUrl: new URL(operatorPrivateConfig.privacyPolicyUrl)
};
addOperatorApi(
  crtoOneOperatorApp,
  identity,
  ...
```

Reply:
Yes, or something like this:

```typescript
addOperatorApi(
  crtoOneOperatorApp,
  {
  name: crtoOneOperatorConfig.name,
  currentPublicKey: operatorPrivateConfig.currentPublicKey,
  dpoEmailAddress: operatorPrivateConfig.dpoEmailAddress,
  privacyPolicyUrl: new URL(operatorPrivateConfig.privacyPolicyUrl)
} : Omit<Identity, 'type'>,
  ...

```

Reply:
This won't compile 😁

If you want to do it inline, you might do this:
```typescript
addOperatorApi(
  crtoOneOperatorApp.app,
  <Omit<Identity, 'type'>>{
    name: crtoOneOperatorConfig.name,
    currentPublicKey: operatorPrivateConfig.currentPublicKey,
    dpoEmailAddress: operatorPrivateConfig.dpoEmailAddress,
    privacyPolicyUrl: new URL(operatorPrivateConfig.privacyPolicyUrl),
  },
```

or 
```typescript
addOperatorApi(
  crtoOneOperatorApp.app,
  {
    name: crtoOneOperatorConfig.name,
    currentPublicKey: operatorPrivateConfig.currentPublicKey,
    dpoEmailAddress: operatorPrivateConfig.dpoEmailAddress,
    privacyPolicyUrl: new URL(operatorPrivateConfig.privacyPolicyUrl),
  } as Omit<Identity, 'type'>,
```

Also I must say I'm quite sceptical as:
- type is already checked automatically by the transpiler so this would be redundant
- the IDE itself helps with intelisence:
 <img width="395" alt="image" src="https://user-images.githubusercontent.com/2690636/168075902-3afafe07-8581-4689-bdba-49b683ce04f5.png">
- why explicitly repeating the type here and not for other method calls (which I don't recommend 😉)

In other words, I don't really understand your comment about "typing for developers, not just for compiler": I admit this method call is relatively verbose, but I don't see how adding extra type declarations would help?